openslides/core/models.py

```diff
@@ -294,6 +297,76 @@ def __str__(self):
         return 'Message {}'.format(self.timestamp)
 
 
+class Message(RESTModelMixin, models.Model):
```

Comment:
Please rename this model to ProjectorMessage. Also rename other classes respectivly.